### ISSUE TITLE
Empty lists trigger a warning

### ISSUE THREAD

**Issue by adamdelman** [Intsights/flake8-intsights#5]
The following code triggers incorrect warning: 
I045 each subentry opener must be in its own line
```python
dict_obj = {
    'stuff': [
        [],
    ],
}
```

**Comment by wavenator** [comment]
do you run an updated version of the linter?
I could not reproduce
